### PR TITLE
V1.0.4

### PR DESCRIPTION
**Pull request by estebanmino**
**Description**

_Write a short description of the changes included in this pull request._

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

**Review by andrepimenta — APPROVED** [review]

File: app/components/Views/ErrorBoundary/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { colors, fontStyles } from '../../../styles/common';
 import { ScrollView } from 'react-native-gesture-handler';
 import Clipboard from '@react-native-community/clipboard';
 import { strings } from '../../../../locales/i18n';
+import Icon from 'react-native-vector-icons/FontAwesome';
```
Comment (nicc938) [inline app/components/Views/ErrorBoundary/index.js:10]:
```suggestion
import Icon from 'react-native-vector-icons/FontAwesome';
```

File: locales/en.json [hunk 2]
```diff
@@ -1323,21 +1323,19 @@
 		"action": "Learn more"
 	},
 	"error_screen": {
-		"title": "We're sorry",
-		"subtitle": "Information can't be shown due to an error.",
+		"title": "An error occured",
```
Comment (nicc938) [inline locales/en.json:1326]:
```suggestion
		"title": "An error occured",
```

Comment (nicc938) [inline locales/en.json:1326]:
```suggestion
		"title": "An error occured",
```

File: locales/en.json [hunk 3]
```diff
@@ -1323,21 +1323,19 @@
...
+		"subtitle": "Your information can't be shown. Don’t worry, your wallet and funds are safe.",
```
Comment (nicc938) [inline locales/en.json:1327]:
```suggestion
		"subtitle": "Your information can't be shown. Don’t worry, your wallet and funds are safe.",
```

**Review by nicc938 — COMMENTED** [review]
screenshoteasy (3)

**Review by ibrahimtaveras00 — APPROVED** [review]
Error page looks good on both OS's 👍

**Pull request closed without merging**